tests/phpunit.bootstrap.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-(function(): void {
+(function (): void {
     require_once __DIR__ . '/../vendor/autoload.php';
-    $secondsToSleep = 3;
+    $secondsToSleep = 30;
```

Comment:
May we can set an ENV var via `phpunit.xml` for that?

Reply:
Im still testing with it, This last commit will be removed when everything is ok.

At the moment theres an issue with running Docker inside scrutinizer. The container where the PHP code runs can not reach the container with rabbitmq... so the tests fail. :(

I already sent a message to them, and Im waiting for some feedback.

Reply:
Ouch! That's okay then :/